appointment-interface/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
```

Comment:
you mean, not TS?

Reply:
Yeah

Reply:
Just not yet:) I will do that later

Reply:
As Donovan told me, it’s better to integrate Typescript as soon as possible because when we build up some logic, it could be harder to migrate to TS
Also, it’ll be easier to track what data you receive and send to prevent any errors in communication between microservices